org.eclipse.jdt.core/.project

```diff
@@ -1,34 +1,25 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>org.eclipse.jdt.core</name>
+	<name>shadows.org.eclipse.jdt.core</name>
 	<comment></comment>
 	<projects>
 	</projects>
 	<buildSpec>
 		<buildCommand>
-			<name>org.eclipse.jdt.core.javabuilder</name>
-			<arguments>
-			</arguments>
-		</buildCommand>
-		<buildCommand>
-			<name>org.eclipse.pde.ManifestBuilder</name>
-			<arguments>
-			</arguments>
-		</buildCommand>
-		<buildCommand>
-			<name>org.eclipse.pde.SchemaBuilder</name>
-			<arguments>
-			</arguments>
-		</buildCommand>
-		<buildCommand>
-			<name>org.eclipse.pde.api.tools.apiAnalysisBuilder</name>
+			<name>org.eclipse.ajdt.core.ajbuilder</name>
 			<arguments>
 			</arguments>
 		</buildCommand>
 	</buildSpec>
 	<natures>
+		<nature>org.eclipse.ajdt.ui.ajnature</nature>
 		<nature>org.eclipse.jdt.core.javanature</nature>
-		<nature>org.eclipse.pde.PluginNature</nature>
-		<nature>org.eclipse.pde.api.tools.apiAnalysisNature</nature>
 	</natures>
+	<linkedResources>
+		<link>
+			<name>apt</name>
+			<type>2</type>
+			<locationURI>WORKSPACE_LOC/eclipse.jdt.core/org.eclipse.jdt.compiler.apt/src</locationURI>
```

Comment:
Like I said above, I did this in order to keep it working in Eclipse like before because you cannot just link to a directory outside of the module path with `../foo/bar` in Eclipse. I explained this in an e-mail already. The only way to make it work inside Eclipse is to use a virtual folder resource like this. Maven does not need it, but Eclipse does.